packages/connect-agreement/src/models/Agreement.ts

```diff
@@ -69,36 +71,40 @@ export default class Agreement {
     )
   }
 
+  async disputeFees(versionId: string): Promise<any> {
+    const version = await this.#connector.version(versionId)
+    const arbitrator = new Contract(version.arbitrator, arbitratorAbi, this.#app.provider)
```

Comment:
Nice, I think this is quite clean. The only thing is the `arbitratorAbi` that we can eventually consider to move it to a Disputables utility package.